### ISSUE TITLE
Video network ACLs

### ISSUE THREAD

**Issue by gerryd** [FOSDEM/infrastructure#101]
For the video network (`185.175.218.0/24`), I think we want to have these ACLs set:

- Inbound:
  - default policy: deny
  - permit all from `51.15.42.23/32` and `2001:bc8:4700:2300::19:f05/128` (control0.video.fosdem.org)
- Outbound:
  - default policy: permit

@krokodilerian, @markvdb: does this look sane to you?

**Comment by krokodilerian** [comment]
@gerryd yes, there is no other traffic going back in. (and this way, control can be used as a jump host back in the video vlan, in worst case scenarios).

**Comment by markvdb** [comment]
+1.
